### DISCUSSION TITLE
Dealing with dropdown (matching HTML and a hidden input) and objects instead of string values

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1722] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)]
Hello,
how to initialize a dropdown build with matching HTML and a hidden input and passing objects as selectable values?
I would like to handle objects like this

```
{
    id: 1,
    name: 'Foo',
    category: 'Bar'
}
```

...and show `name` property in the value but get the `id` in the selected item. How to reach that?
Seems I can only pass string to `data-value` attribute of HTML...

**Comment by ko2in** [comment]
Is [this](https://jsfiddle.net/ko2in/ym835r0x/4/) what you're looking for?
